engine/src/main/java/org/terasology/engine/modes/StateMainMenu.java

```diff
@@ -43,134 +44,140 @@
 import org.terasology.utilities.Assets;
 
 /**
- * The class implements the main game menu.
- * <br><br>
+ * The class implements the main game menu. <br>
```

Comment:
Not sure why this got moved but a better option would be deleting both `<br>` tags and the `@version` thing, not very useful. Just need the single line with a . at the end :-)